src/effects/builtin/whitenoiseeffect.cpp

```diff
@@ -0,0 +1,76 @@
+#include "effects/builtin/whitenoiseeffect.h"
+
+#include <QRandomGenerator>
+
+#include "util/rampingvalue.h"
+
+// static
+QString WhiteNoiseEffect::getId() {
+    return QStringLiteral("org.mixxx.effects.whitenoise");
+}
+
+// static
+EffectManifestPointer WhiteNoiseEffect::getManifest() {
+    EffectManifestPointer pManifest(new EffectManifest());
+    pManifest->setId(getId());
+    pManifest->setName(QObject::tr("White Noise"));
+    pManifest->setAuthor("The Mixxx Team");
+    pManifest->setVersion("1.0");
+    pManifest->setDescription(QObject::tr("Creates active white noise sound"));
+
+    // This is loudness parameter
+    EffectManifestParameterPointer intensity = pManifest->addParameter();
+    intensity->setId("Dry/Wet");
```

Comment:
conventionally we use lowercase for the ID strings
```suggestion
    intensity->setId("dry_wet");
```